go/cmd/dolt/commands/merge.go

```diff
@@ -102,7 +102,7 @@ func (cmd MergeCmd) Exec(ctx context.Context, commandStr string, args []string,
 	}
 
 	if _, ok := queryist.(*engine.SqlEngine); !ok {
-		cli.PrintErrln(cli.RemoteUnsupportedMsg)
+		cli.Println(cli.RemoteUnsupportedMsg)
```

Comment:
For this, you need to stick commandStr at the beginning. If you want to standardize this usage, this is the other example: https://github.com/dolthub/dolt/blob/1901dfeae2566a291e308a480802abc0339cdfcc/go/cmd/dolt/commands/checkout.go#L104